encoder.py

```diff
@@ -48,7 +48,21 @@ def calc(MSG, hash_type):
 
         result = [str(bin(ord(elem))[2:]) for elem in MSG]
         return ' '.join(result)
-
+    
+     elif hash_type == 'book':
+        """Convert message to key representation"""
+        
+        passage = click.prompt("Enter passage", type=str)
+        w_l=passage.split(' ')
```

Comment:
Follow spacing conventions as can seen in the rest of the code. You've used spaces correctly on Line 55 (above).
Spaces before and after for assignment, no spaces if written within a function call.

Reply:
encoder.py has been updated accordingly.